app/validation/messages.py

```diff
@@ -0,0 +1,13 @@
+from flask.ext.babel import gettext as _
+
+# Set up default error and warning messages
```

Comment:
I've renamed this module to error_messages.  It provides a central place for error messages that can easily be localised